### PR TITLE
expose opa version as runtime information

### PR DESCRIPTION
**Pull request by omerlh**
Exposed the runtime OPA version on `opa.runtime` - can be used in discovery, for example, to expose features based on OPA version (e.g. multi bundle is supported only from a specific version).

### CODE REVIEW

File: version/version.go [hunk 1]
```diff
@@ -8,7 +8,7 @@ package version
 // Version information that is displayed by the "version" command and used to
 // identify the version of running instances of OPA.
 var (
-	Version   = ""
+	Version   = "1.0.4"
```
Comment (srenatus) [inline version/version.go]: ❓ What is this about? IIRC, the proper version is set at buildtime here. 🤔

Comment (omerlh) [reply version/version.go]: By accident, pushing a fix - sorry

**Comment by srenatus** [comment]
@omerlh I'm afraid you've gotten it all reverted now? 😉

**Comment by omerlh** [comment]
ahh damn, too much git magic 😉 now it should be fine

**Comment by omerlh** [comment]
Added, it was on the original PR and somehow lost 🤷‍♂

**Pull request merged**